packages/fxa-auth-server/scripts/check-users.js

```diff
@@ -0,0 +1,217 @@
+
+function sanitizeValue(v) {
+  if (v === undefined || v === null) {
+    return '';
+  }
+  return v;
+}
+class User {
+  constructor(email, password, db) {
+    this.email = email;
+    this.password = password;
+    this.db = db;
+  }
+
+  // FxA uses HKDF to derive the authPW from the user's password.
+  // This is typically done on the client side since FxA never sees the user's
+  // cleartext password.
+  async getCredentials(email, password) {
+    const stretch = await pbkdf2.derive(
+      Buffer.from(password),
+      hkdf.KWE('quickStretch', email),
+      1000,
+      32
+    );
+    this.authPW = await hkdf(stretch, 'authPW', null, 32);
+    this.unwrapBKey = await hkdf(stretch, 'unwrapBKey', null, 32);
+    return this;
+  }
+
+  async stats() {
+    try {
+      const accountRecord = await this.db.accountRecord(this.email);
+      const credentials = await this.getCredentials(
+        accountRecord.primaryEmail.normalizedEmail,
+        this.password
+      );
+
+      // Check the user password against the stored hash in DB
+      const password = new Password(
+        credentials.authPW,
+        accountRecord.authSalt,
+        accountRecord.verifierVersion
+      );
+      const verifyHash = await password.verifyHash();
+      const passwordMatch = await this.db.checkPassword(
+        accountRecord.uid,
+        verifyHash
+      );
+
+      // Check to see if user has MFA enabled
+      let mfaEnabled = false;
+      try {
+        const totpToken = await this.db.totpToken(accountRecord.uid);
+        if (totpToken) {
+          mfaEnabled = true;
+        }
+      } catch (err) {}
+
+      const s = {
+        email: this.email,
+        exists: true,
+        passwordMatch,
+        mfaEnabled,
+        keysChangedAt: accountRecord.keysChangedAt,
+        profileChangedAt: accountRecord.profileChangedAt,
+        hasSecondaryEmails: accountRecord.emails.length > 1,
+        isPrimaryEmailVerified: accountRecord.primaryEmail.isVerified,
+      };
+      const stat = `${s.email},${s.exists},${sanitizeValue(
+        s.passwordMatch
+      )},${sanitizeValue(s.mfaEnabled)},${sanitizeValue(
+        s.keysChangedAt
+      )},${sanitizeValue(s.profileChangedAt)},${sanitizeValue(
+        s.hasSecondaryEmails
+      )},${sanitizeValue(s.isPrimaryEmailVerified)}`;
+
+      console.log(stat);
+
+      return s;
+    } catch (err) {
+      return {
+        exists: false,
+        email: this.email,
+      };
+    }
+  }
+}
+
+class CheckUsers {
+  constructor(filename) {
+    this.users = [];
+    this.db = undefined;
+    this.filename = filename;
+  }
+
+  getItems() {
+    try {
+      const input = fs
+        .readFileSync(path.resolve(this.filename))
+        .toString('utf8');
+
+      if (!input.length) {
+        return [];
+      }
+
+      // Parse the input file CSV style
+      return input.split(/\n/).map((s) => {
+        const delimiter = program.delimiter || ':';
```

Comment:
nit: not sure it matters, but you should be able to specify the default option values in the commander config above on L23 directly like this (per https://www.npmjs.com/package/commander#default-option-value):

```diff
-.option('-d, --delimiter [delimiter]', 'Delimiter for input file')
+.option('-d, --delimiter [delimiter]', 'Delimiter for input file', ':')
```